model/src/main/java/com/epam/esm/model/service/converter/impl/CertificateConverter.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.esm.model.service.converter.impl;
+
+import com.epam.esm.model.dao.entity.GiftCertificate;
+import com.epam.esm.model.service.converter.Converter;
+import com.epam.esm.model.service.dto.CertificateDTO;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Component
+public class CertificateConverter implements Converter<CertificateDTO, GiftCertificate> {
```

Comment:
I think that it's Mapper not a Converter

Reply:
renamed